components/api/src/main/java/com/hotels/styx/api/StreamingHttpMessage.java

```diff
@@ -93,4 +95,21 @@ default boolean chunked() {
         return HttpMessageSupport.chunked(headers());
     }
 
+    /**
+     * Consume the message by discarding the message body.
+     * <p>
+     * This method reads the entire message body from the networks and black holes
+     * all the traffic. This has the benefit of keeping the underlying TCP connection
+     * open for connection pooling.
+     * <p>
+     */
+    default void consume() {
+        body().drop().aggregate(1)
```

Comment:
I don't find this method too clear either.  Also, will the aggregate() complete exceptionally and need to generate a stack trace?  I don't have a suggestion on how to improve it though...

Reply:
Okay. I will take this comment onboard and we will do a separate round of improvements.